DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "SymantecDLP",
+    "title": "Symantec DLP",
+    "publisher": "Broadcom",
```

Comment:
Publisher name should be part of id and title. Please refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Data%20Connectors%20Template%20Guidance.md for guidelines.

Reply:
changes have been pushed